src/protocols/bdx/tests/TestBdxTransferSession.cpp

```diff
@@ -129,8 +129,13 @@ void VerifyStatusReport(nlTestSuite * inSuite, void * inContext, const System::P
     NL_TEST_ASSERT(inSuite, err == CHIP_NO_ERROR);
     NL_TEST_ASSERT(inSuite, payloadHeader.GetProtocolID() == Protocols::kProtocol_Protocol_Common);
     NL_TEST_ASSERT(inSuite, payloadHeader.GetMessageType() == static_cast<uint8_t>(Protocols::Common::MsgType::StatusReport));
+    if (headerSize > msg->DataLength())
```

Comment:
In theory, this should have caused `err` to be an error....  I'm OK with keeping it, or ok with a comment saying we can assume `headerSize` is small enough because `err == CHIP_NO_ERROR` here.

Certainly non-test code makes that assumption.